tests/auto/rt_auto.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash --login
+set -eux
+
+export RT_COMPILER='intel'
```

Comment:
I see 'intel' is hard-coded here. Do I still need to login to Hera and Cheyenne and manually run rt.sh for gnu compiler?

Reply:
for now, yes. Thats on the list for the next version of the code. I want to get a first version out so that everyone can see how it works now and we can build a set of idea to shape the final version of the code.